multicluster/controllers/multicluster/member/clusterset_controller_test.go

```diff
@@ -50,7 +50,32 @@ func TestMemberClusterDelete(t *testing.T) {
 	reconciler := MemberClusterSetReconciler{
 		Client:           fakeClient,
 		remoteCommonArea: commonArea,
+		clusterSetID:     common.ClusterSetID("clusterset1"),
 	}
+
+	// Delete a different ClusterSet.
+	if _, err := reconciler.Reconcile(common.TestCtx, reconcile.Request{
+		NamespacedName: types.NamespacedName{
+			Namespace: "default",
+			Name:      "clusterset2",
+		}}); err != nil {
+		t.Errorf("Member ClusterSet Reconciler should handle delete event successfully but got error = %v", err)
+	} else {
+		memberClusterAnnounce := &mcv1alpha1.MemberClusterAnnounce{}
+		err := fakeRemoteClient.Get(common.TestCtx, types.NamespacedName{
+			Namespace: "default",
+			Name:      "member-announce-from-cluster-a",
```

Comment:
@luolanzone I feel better to remove "member-announce-from-" in the memberClusterAnnounce name. Of course this is not urgent.